runtime/rococo/src/lib.rs

```diff
@@ -789,6 +791,40 @@ impl mmr_common::Config for Runtime {
 	type ParachainHeads = Paras;
 }
 
+parameter_types! {
+	// This is a pretty unscientific cap.
+	//
+	// Note that once this is hit the pallet will essentially throttle incoming requests down to one
+	// call per block.
+	pub const MaxRequests: u32 = 50;
```

Comment:
```suggestion
	pub const MaxRequests: u32 = 4 * HOURS as u32;
```

Can we do something like that instead? This will allow us to catch-up immediately in case of 4 hours downtime.